frontend/src/components/common/form/InputGroup.tsx

```diff
@@ -66,9 +65,6 @@ export const InputGroup: React.FC<InputGroupProps> = ({
   displayErrorTooltips,
   ...rest
 }) => {
-  const error = getIn(formikProps?.errors, field);
```

Comment:
rather then duplicating out the error tooltip logic here, just pass the displayerrortooltips flag to the underlying input component.